### PR TITLE
Docker usage: focus on using bats isolated in other project

### PR DESCRIPTION
**Pull request by arve0**
Focus on how to use bats in another project.

Document how one loads libraries, solves #707.

- [x] I have reviewed the [Contributor Guidelines][contributor].
- [x] I have reviewed the [Code of Conduct][coc] and agree to abide by it

[contributor]: https://github.com/bats-core/bats-core/blob/master/docs/CONTRIBUTING.md
[coc]:         https://github.com/bats-core/bats-core/blob/master/docs/CODE_OF_CONDUCT.md

### CODE REVIEW

File: docs/source/docker-usage.md [hunk 1]
```diff
@@ -2,11 +2,29 @@
...
   
 ## Basic Usage
 
+For test suites that are intended to run in isolation from the bats project, you can mount the test directory and run the [official bats docker image](https://hub.docker.com/r/bats/bats):
+
+```bash
+$ docker run -it -v "$PWD:/code" bats/bats:latest /code/test
+```
+
+Docker image includes libaries like [bats-support](https://github.com/bats-core/bats-support) and [bats-assert](https://github.com/bats-core/bats-assert), which can be loaded in `setup` like this:
```
Comment (martin-schulze-vireso) [inline docs/source/docker-usage.md:17]:
```suggestion
This Docker image includes libaries like [bats-support](https://github.com/bats-core/bats-support) and [bats-assert](https://github.com/bats-core/bats-assert), which can be loaded in `setup` like this:
```

File: docs/source/docker-usage.md [hunk 2]
```diff
@@ -2,11 +2,29 @@
 
 - [Docker Usage Guide](#docker-usage-guide)
   * [Basic Usage](#basic-usage)
+  * [Basic Usage for bats project](#basic-usage-for-bats-project)
   * [Docker Gotchas](#docker-gotchas)
   * [Extending from the base image](#extending-from-the-base-image)
   
 ## Basic Usage
 
+For test suites that are intended to run in isolation from the bats project, you can mount the test directory and run the [official bats docker image](https://hub.docker.com/r/bats/bats):
```
Comment (martin-schulze-vireso) [inline docs/source/docker-usage.md:11]: In comparison to the original, this Changes the meaning. Yes, we want to document how to run tests in projects that are not Bats with the docker file. However, the text was referring to the third party code that should be tested.

```suggestion
For test suites that are intended to run in isolation from their project code, you can mount the test directory and run the [official bats docker image](https://hub.docker.com/r/bats/bats):
```

**Review by martin-schulze-vireso — APPROVED** [review]

**Comment by arve0** [comment]
@martin-schulze-vireso Need anything more from me to get this merged?

**Comment by martin-schulze-vireso** [comment]
Seeing the test failure: you could add a changelog entry.

**Comment by arve0** [comment]
> Seeing the test failure: you could add a changelog entry.

Fixed.

**Comment by martin-schulze-vireso** [comment]
Thanks for your contribution.

**Pull request merged**